model_checking/src/lib.rs

```diff
@@ -38,3 +41,49 @@ pub fn wfi() {
         "wfi instruction emulation is not correct"
     );
 }
+
+#[cfg_attr(kani, kani::proof)]
+#[cfg_attr(test, test)]
+pub fn interrupt_virtualization() {
+    let (mut ctx, _, mut sail_ctx) = symbolic::new_symbolic_contexts();
+
+    // To avoid an overflow, we artificially set a max limit size for mtvec
+    ctx.csr.mtvec = min(ctx.csr.mtvec, usize::MAX - 10000);
+    sail_ctx.mtvec = BitField::new(min(sail_ctx.mtvec.bits.bits(), (usize::MAX - 10000) as u64));
```

Comment:
Did one of the two implementation get a fault? I am wondering what should be the right behavior, but most likely wrapping add? If the sail implementation does it correctly then we should fix our code :)

Reply:
Sail uses a wrapping add, so I do the same now  ^^ Fixed